packages/x-codemod/README.md

```diff
@@ -119,7 +139,26 @@ npx @mui/x-codemod v7.0.0/data-grid/preset-safe <path|folder>
 
 The list includes these transformers
 
-NO CODEMOD ADDED YET
+- [`rename-components-to-slots-data-grid`](#rename-components-to-slots-data-grid)
+
+#### `rename-components-to-slots-data-grid`
+
+Renames the `components` and `componentsProps` props to `slots` and `slotProps`, respectively.
+
+This change only affects data grid components.
```

Comment:
```suggestion
This change only affects the Data Grid components.
```

Reply:
I'm using `Grid` because Sam used it in #7127 :+1:

Reply:
IMO `Grid` might be confused with material-ui [Grid](https://mui.com/material-ui/react-grid/) by some users. `Data Grid` seems more _directed_. 🤷‍♂️

Reply:
Indeed, I can only second this point. 👍 💯